distributed/client.py

```diff
@@ -3319,7 +3316,7 @@ def call_stack(self, futures=None, keys=None):
         keys = keys or []
         if futures is not None:
             futures = self.futures_of(futures)
-            keys += list(map(tokey, {f.key for f in futures}))
+            keys.extend(list(map(tokey, {f.key for f in futures})))
```

Comment:
You should be able to just pass an iterable here so you shouldn't have to wrap what's passed to `.extend(...)` in a list.